app/lib/logic/provider_signup.rb

```diff
@@ -256,7 +303,7 @@ def create_default_account_plan!
       end
 
       def create_default_service_plan!
-        plan = service_plans.find_by_name('Default') || service_plans.create!(name: 'Default')
+        plan = service_plans.find_by(name: 'Default') || service_plans.create!(name: 'Default')
```

Comment:
isn't `find_by_name` quicker?

Reply:
No and it is deprecated so better use `find_by`